dev-server.mjs

```diff
@@ -0,0 +1,1032 @@
+// Dependencies
+import * as http from 'node:http'
+import * as filesystem from 'node:fs/promises'
+
+const server = http.createServer()
+server.on('request', async (request, response) => {
+	try {
+		const urlPath = (request.url.endsWith('/')) ? `${request.url}index.html` : request.url
```

Comment:
this can be undefined, and then it tries to loop for `./appundefined` afterwards